src/main/scala/uk/gov/nationalarchives/draftmetadatavalidator/CSVHandler.scala

```diff
@@ -13,14 +13,14 @@ class CSVHandler {
     val allRowsWithHeader = reader.all()
     val fileRows = allRowsWithHeader match {
       case _ :: rows =>
-        rows.map { case fileName :: data =>
+        rows.map { case _ :: _ :: _ :: data =>
           FileRow(
```

Comment:
Named arguments here would improve readability, took a little while to work out what was going on here.

Have we considered a csv library of some sort here? Realise I'm jumping into this conversation late and it might be outside the scope of this iteration, but this seems quite brittle- if the position of UUID changed or any columns were added afterwards this would no longer work. We use this one in TRE https://github.com/tototoshi/scala-csv

Also usually keen to avoid `.last`, ideally could we integrate .lastOption with some sort of error state if we want to hard fail if it's not there?